data/assets/gui/images.asset

```diff
@@ -0,0 +1,48 @@
+-- This asset requires OpenSpace to be built with the OPENSPACE_MODULE_SYNC enabled
+
+local assetHelper = asset.require("util/asset_helper")
+
+asset.syncedResource({
+    Type = "UrlSynchronization",
+    Name = "Icons",
+    Identifier = "planet_icons",
+    Url = {
+        "http://data.openspaceproject.com/dist/sci/mastertheses/2018/icons/solarsystem/earth.png",
```

Comment:
We may want to move this before merging?

Reply:
@alexanderbock Can put it on the server maybe?

Reply:
It's moved and the URLs are updated